README.md

```diff
@@ -1,62 +1,84 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# FoodMD
 
 > Description the project.
-
+The FoodMD is a web application that has a list of foods and recipes to prepare them. The web application was built based on two APi's, which are the Meals DB and the Involvement API. The web app makes APi call to render a list of meals and also make another API call when a user likes or comment.
 
 ## Built With
-
 - Major languages
+HTML
+JavaScript -Webpack -JS Modules
+CSS
 - Frameworks
+None
 - Technologies used
+-Git
+-Jest(for testing)
+-Webpack
 
-## Live Demo (if available)
+## Live Demo 
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://capable-tarsier-080db1.netlify.app/)
 
+## Demo Video from Authors
+[Video](https://drive.google.com/file/d/1A7BDcPe5crGGqIxkneWKVb0rq3Cw2CRG/view?usp=sharing)
```

Comment:
- [ ] @team good work in your presentation, however you forgot to indicate what your application is about and also the special feature or part of code that you found interesting kindly record another video to accomplish the missing parts.